### PR TITLE
User Guide: add FAQ section to address common problems #297

### LINKED ISSUE #297: User Guide: add FAQ section to address common problems
E.g. 
- _How does formats and ignore glob list works?_
- _Does RepoSense work on private repositories?_ 

This helps to resolve some TLDR issues.

### PR DESCRIPTION
**Pull request by yamidark**
Fixes #297 

```
Our UserGuide does not have a FAQ section.

Let's add a FAQ section in our UserGuide to address common questions
and issues, allowing users troubleshoot them themselves.
```

### CODE REVIEW

**Comment by yamidark** [comment]
@reposense/devs For review of the current few FAQs first, will continue to look into any more others to add in. Do let me know if I should add any else in as well.

File: docs/UserGuide.md [hunk 1]
```diff
@@ -302,5 +303,25 @@ Alternatively, you can use csv config files to customize the analysis as before
...
+
+**Q:** My commit contributions does not appear in the ramp chart (despite appearing in the contribution bar and code panel)?<br>
+**A:** This is probably a case of giving an incorrect author name alias in your `author-config.csv` file.
+Please refer to [A Note About Git Author Name](#a-note-about-git-author-name) above on how to find out the correct author name you are using, and how to change it.
+Also ensure that you have added all author name aliases that you may be using.
+
+**Q:** I have added/edited the standalone config file in my local repository, but RepoSense is not using it when running the analysis?<br>
+**A:** Ensure that you have committed the changes to your standalone config file first before running the analysis, as RepoSense is unable to detect uncommitted changes to your local repository.
+
+**Q:** I am able to run RepoSense on my repository on a Linux/Mac OS, but it fails on a Windows OS?
```
Comment (eugenepeh) [inline docs/UserGuide.md]: need a `<br/>` after this line

**Comment by yamidark** [comment]
@reposense/devs For next round of review

**Comment by yamidark** [comment]
@yong24s Not exactly sure which question is more likely to occur over the other 🤔.
Perhaps @eugenepeh can tell us his experience in last semester CS2103/CS2113? 😅

**Pull request merged**